### ISSUE TITLE
MDX hook is buggy

### ISSUE THREAD

**Issue by phated** [gulpjs/interpret#92]
It seems that the deasync module causes pretty frequent failures in our CI. I'll try to ask @wooorm about it.

**Comment by wooorm** [comment]
Yeah. deasync is buggy. It’s only used in the deprecated CJS hook. Perhaps you can use the beta node loader instead? 🤷‍♂️
